Components/lists/lists.css

```diff
@@ -0,0 +1,47 @@
+/* simple list */
+
+.container-list {
+  list-style-type: none;
+  background-color: var(--card-container-bg);
+  color: var(--text-color);
+  width: 30rem;
+  padding: 1rem;
+  line-height: 3rem;
+  text-align: left;
+  border-radius: 0.5rem;
+}
+
+.list-title {
+  border-bottom: 1px solid var(--text-color);
+  padding: 1.5rem;
+  margin-bottom: 1rem;
+}
+
+.container-list li {
+  margin-left: 2rem;
+  cursor: pointer;
+}
+
+.container-list li:hover {
+  color: var(--primary-color);
+  transition: 0.2s ease-in-out;
+}
+
+/* notification list */
+
+.notification-list {
+  list-style-type: none;
+  background-color: var(--card-container-bg);
+  width: 30rem;
+  color: var(--text-color);
+  line-height: 3rem;
+  border-radius: 0.5rem;
+}
+
+.notification-list li {
+  padding: 1rem 1.5rem;
+  margin: 1rem 1rem;
+  border: 1px solid #e6e6e9;
+  border-radius: 0.5rem;
+  background-color: #f7f5f5;
+}
```

Comment:
You can use CSS variables for colors to be consistent.

Reply:
thanks for that! actually this is the only place i am using this colors so used hex instead.

Reply:
Okay, Everything else looks good.